### PR TITLE
Group proofread 2022-03-19

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 8/08_00_ARC08_8.txt [hunk 1]
```diff
@@ -430,6 +430,7 @@ The figure's laugh echoes through the darkness.
 -- Page 507, Offset 21032.
 -- ぎょろりと血走ったその瞳が“ようやく気が付いたのか”と嘲笑している。
 // Dude can you see him or not
+// group pr note: eye/eyes?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 8/08_00_ARC08_8.txt:433]: I think that plural is OK here, it initially only mentions one visible eye but there's an animation that plays between that and now where 2 eyes are visible. Plus, if it's SHIKI+Roa, that sprite has 2 eyes one is just covered somewhat by hair

File: script/Arcueid/Day 8/08_00_ARC08_8_1B.txt [hunk 2]
```diff
@@ -140,6 +140,7 @@ Quality of her explanation aside, what she's talking about does ring a bell.
 [sha:2910cc7549526b9a3e0bf95f405450b33c03256d]{
 -- Page 553, Offset 21165.
 -- 志貴クンは無神論者でしょ？　だから実感は湧かないだろうけど、私たち教会は主の御名のもと、人の善き営みを守る意思がある。たとえそれが、こんな末法の国でもね」
+// group pr note: something about Noel voicing her displeasure to Japan, country about to fall soon or something
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_1B.txt:143]: Like I said in group PR: I feel pretty strongly that stripping this of its literal meaning is to take away a genuine allusion.

Petrikow seems to think that the allusion's meaning is all that matters and that it should just be "Even in a country as degenerate as this one" or something to that effect

File: script/Arcueid/Day 8/08_00_ARC08_8_2.txt [hunk 3]
```diff
@@ -246,6 +247,7 @@ And not even a very serious question at that.
 [sha:6fdfb403ceb9f85866603e5ce884daada1fe977c]{
 -- Page 598, Offset 21287.
 -- なに、コドモのくせにカマかけてきたワケ？　そういうところはナマイキで、先生、キライだな！」
+// group pr note:change brat?
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_2.txt:250]: What, even though you're just some brat you're trying to ask %{i}me%{/i} leading questions?

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 8/08_00_ARC08_8_2.txt [hunk 4]
```diff
@@ -33,6 +33,7 @@ Oh I see, you're so afraid that you're frozen solid ♥
 [sha:60f28a7e01b746fb32af034f54cecfbd12c2d17b]{
 -- Page 583, Offset 21247.
 -- うんうん、やっぱり文系の美少年はそうでなくっちゃ！
+// group pr note: change humanities
 Mmmhmm, that's just how a handsome young man in the humanities should behave!
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_8_2.txt:37]: in the %{i}liberal arts%{/i}

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 8/08_00_ARC08_3_1.txt [hunk 5]
```diff
@@ -184,12 +184,12 @@ I didn't want to waste your time again like yester―――"
 -- Page 172, Offset 20096.
 -- 「それこそ余分な気遣いだっての。協力すると言った以上、どんな作業だって一緒にやるんだから夜まで待っててくれよ。
 "I'm telling you there's no need to go that far.
- I said I'd team up with you, so whatever needs doing just hold on until tonight then let's do it together, okay?
+ I said I'd team up with you, so whatever needs doing just hold on until tonight, then let's do it together, okay?
```
Comment (yk1512) [inline script/Arcueid/Day 8/08_00_ARC08_3_1.txt:187]: I said I'd team up with you so just hold on until tonight. We can settle whatever needs to be done together, okay?

**Review by yk1512 — APPROVED** [review]

**Pull request merged**